IdentityCore/src/controllers/MSIDLocalInteractiveController.m

```diff
@@ -66,41 +66,54 @@ - (nullable instancetype)initWithInteractiveRequestParameters:(nonnull MSIDInter
 
 - (void)acquireToken:(MSIDRequestCompletionBlock)completionBlock
 {
+    MSID_LOG_INFO(self.requestParameters, @"Beginning interactive flow.");
+    
     if (!completionBlock)
     {
-        MSID_LOG_ERROR(nil, @"Passed nil completionBlock");
+        MSID_LOG_ERROR(self.requestParameters, @"Passed nil completionBlock. Interactive flow finished.");
         return;
     }
 
     [[MSIDTelemetry sharedInstance] startEvent:self.interactiveRequestParamaters.telemetryRequestId eventName:MSID_TELEMETRY_EVENT_API_EVENT];
 
     MSIDInteractiveTokenRequest *interactiveRequest = [self.tokenRequestProvider interactiveTokenRequestWithParameters:self.interactiveRequestParamaters];
 
-    [interactiveRequest executeRequestWithCompletion:^(MSIDTokenResult * _Nullable result, NSError * _Nullable error, MSIDWebMSAuthResponse * _Nullable msauthResponse) {
-
+    [interactiveRequest executeRequestWithCompletion:^(MSIDTokenResult * _Nullable result, NSError * _Nullable error, MSIDWebMSAuthResponse * _Nullable msauthResponse)
+    {
+        MSIDRequestCompletionBlock completionBlockWrapper = ^(MSIDTokenResult * _Nullable result, NSError * _Nullable error)
```

Comment:
Can't we just log this line in completion block of executeRequestWithCompletion before we call anything else? Result or error don't get changed.

Reply:
If we go inside `executeRequestWithCompletion` there also few places where we call completion block, so I think it is the best way to log in wrapper block so we don't duplicate code. Would could be an issue with current logic?

Reply:
As we dicussed offline, we will keep current logic.